_sass/_base.scss

```diff
@@ -99,6 +99,16 @@ p:last-of-type {
 /**
  * Links
  */
+
+button, a {
```

Comment:
![I understood that reference](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExZWlvOW5xbTFxeHNyOGF2MmV5amRvYmN5ZGtrcHZiNzI3cmdyMHI4cCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/tnYri4n2Frnig/giphy.gif)

Reply:
![giphy](https://github.com/wtfluckey/wtfluckey.com/assets/2893463/843a7bcc-138c-4444-83fa-18aba097ec93)